src/wfs/src/test/java/org/geoserver/wfs/outputformat/WFSOutputFormatCallbackTest.java

```diff
@@ -0,0 +1,67 @@
+/* (c) 2022 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wfs.outputformat;
+
+import static org.easymock.EasyMock.createNiceMock;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.replay;
+import static org.junit.Assert.assertEquals;
+
+import java.util.Collections;
+import org.geoserver.config.GeoServer;
+import org.geoserver.ows.Request;
+import org.geoserver.platform.Operation;
+import org.geoserver.platform.Service;
+import org.geoserver.platform.ServiceException;
+import org.geoserver.wfs.WFSInfo;
+import org.junit.Test;
+
+/** Unit test suite for {@link WFSOutputFormatCallback} */
+public class WFSOutputFormatCallbackTest {
```

Comment:
Add a test for capabilities generation too.

Reply:
added